docs/graphql/core/deployment/deployment-guides/nhost-one-click.rst

```diff
@@ -25,36 +25,31 @@ One-click deploy on Nhost
 
 Once you're logged into your Nhost account, click the button bellow to configure your project.
 
-.. image:: https://nhost.io/images/create-project-button.png
-   :width: 300px
-   :class: no-shadow
-   :alt: Deploy with Nhost
+.. thumbnail:: /img/graphql/core/deployment/create-project-nhost.png
+   :alt: Create a project
 
 On the following page you can choose a plan and a location for your project:
 
-.. image:: https://nhost.io/images/create-project-form.png
-   :class: no-shadow
-   :alt: Choose a plan and locatino for your project
+.. thumbnail:: /img/graphql/core/deployment/choose-plan-nhost.png
```

Comment:
Images don't show up on the whole Nhost guide

![image](https://user-images.githubusercontent.com/5722022/97604311-9e707800-1a0d-11eb-90d7-f4d2634e0565.png)

Reply:
oh no, not again.